content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+- If you used a different naming convention for your instance names, you will need to replace 'app-direct-node-0(123)' with your instance names.
+- There are two entries specific to the Application, one for each component (app-direct-frontend/backend & postgres frontend/backend). If you are using a different application, adjust based on the endpoints you want to access through the load balancer. The remaining entries are for the KOTs UI and API and should be left as-is other than the instance names and ip addresses
+
+```shell
+global
+defaults
+   timeout client          60s
+   timeout server          60s
+   timeout connect         60s
+#############################################################
+listen stats
+   bind *:8080
+   mode http
+   stats enable
+   stats uri /
+   stats hide-version
+#############################################################
+frontend app-direct-frontend
+   mode tcp
+   bind    *:80
+   default_backend app-direct-backend
+backend app-direct-backend
+   mode                    tcp
+   balance roundrobin
+   option tcp-check
+   server app-direct-node01 10.240.0.37:80 check
+   server app-direct-node02 10.240.0.39:80 check
+   server app-direct-node03 10.240.0.71:80 check
+#############################################################
+frontend postgres-frontend
+   mode tcp
+   bind    *:5432
+   default_backend postgres-backend
+backend postgres-backend
+   mode                    tcp
+   balance roundrobin
+   option tcp-check
+   server app-direct-node01 10.240.0.37:5432 check
+   server app-direct-node02 10.240.0.39:5432 check
+   server app-direct-node03 10.240.0.71:5432 check
+#############################################################
+frontend replicated-frontend
+   mode tcp
+   bind    *:8800
+   default_backend replicated-backend
+backend replicated-backend
+   mode                    tcp
+   balance roundrobin
+   option tcp-check
+   server app-direct-node01 10.240.0.37:8800 check
+   server app-direct-node02 10.240.0.39:8800 check
+   server app-direct-node03 10.240.0.71:8800 check
+###########################################################
+frontend kube-api-frontend
+   mode tcp
+   bind    *:6443
+   default_backend kube-api-backend
+backend kube-api-backend
+   mode                    tcp
+   balance roundrobin
+   option tcp-check
+   server app-direct-node01 10.240.0.37:6443 check
+   server app-direct-node02 10.240.0.39:6443 check
+   server app-direct-node03 10.240.0.71:6443 check
+
+```
+
+For more details on the structure of the config file, please refer to the HA Proxy [documentation](https://www.haproxy.com/blog/the-four-essential-sections-of-an-haproxy-configuration/)
+
+To copy the config file into the VM, we are going to use scp. The command below will copy it to your home folder on the VM.
+
+```shell
+gcloud compute scp haproxy.cfg app-direct-ha-proxy:~/
+```
+
+Now we install Ha Proxy using apt. 
```

Comment:
I noticed this is sometimes `HA Proxy` and sometimes `Ha Proxy` -- what should we standardize on here?

Reply:
I went through the guide and ensured it consistently refers to it as 'HAProxy'